colcon_ros/task/ament_python/build.py

```diff
@@ -51,8 +51,11 @@ def __init__(self):  # noqa: D107
         setup_py_data = get_setup_data(self.context.pkg, env)
 
         # check if the package index and manifest are being installed
-        data_files = get_data_files_mapping(
-            setup_py_data.get('data_files', []))
```

Comment:
The patch can be simplified by a lot - by just changing this line to: `setup_py_data.get('data_files') or []`.